src/widgets/helper/ChannelView.cpp

```diff
@@ -2080,6 +2080,10 @@ void ChannelView::handleLinkClick(QMouseEvent *event, const Link &link,
             }
         }
         break;
+        case Link::Reconnect: {
+            this->underlyingChannel_.get()->reconnect();
```

Comment:
Can this be used here instead? I did not have a way to test it, but I'm pretty sure it's the same in a nicer form.
```suggestion
            this->channel()->reconnect();
```

Reply:
When I tested it with `this->channel()`, it didn't work.

Reply:
`ChannelView::channel_` is different from `ChannelView::underlyingChannel_` and doesn't seem to be connected to the read/write connections.